### DISCUSSION TITLE
HTML code blocks not displaying properly

### DISCUSSION THREAD

**Discussion by lhorwath** [squidfunk/mkdocs-material#3860] [General]
Hello there, I'm trying to set up a grid for my home page but neither my code or the sample code provided in the documentation displays properly.

For reference this is the code on `index.md`:

```
# DAS Hub

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.

<div class="grid cards" markdown="block">

- :fontawesome-brands-html5: __HTML__ for content and structure
- :fontawesome-brands-js: __JavaScript__ for interactivity
- :fontawesome-brands-css3: __CSS__ for text running out of boxes
- :fontawesome-brands-internet-explorer: __Internet Explorer__ ... huh?

</div>

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```

The page loads and the pipeline runs without any issue, but instead of rendering a grid it displays a list. 

I've enabled the proper extension in my `mkdocs.yml` file:

```
markdown_extensions:
  - attr_list
  - md_in_html
```

I've also tried altering the tag in the `<div class="grid cards" markdown="block">` to either `<div class="grid cards" markdown>` or `<div class="grid cards" markdown="1">`, but to no avail.

Does anyone here have a clue as to what I am doing wrong? I've added a screenshot of the page as it currently renders with the aforementioned code.

<img width="1227" alt="Screenshot" src="https://user-images.githubusercontent.com/35267405/165921382-ab651db3-4d58-47ed-9b3c-c628ac2a7e05.png">

**Comment by squidfunk** [comment]
Thanks for reporting. Are you using the [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/) build? Card grids are currently only available to sponsors of the project:

<img width="714" alt="Bildschirmfoto 2022-04-29 um 12 10 49" src="https://user-images.githubusercontent.com/932156/165925770-ac9f613e-79f6-4242-9ac0-ebaac1da6537.png">

**Reply by lhorwath** [reply]
Oh that might be why. I'd have to check with my manager to see whether or not we are. I assumed we were, but in the event I am wrong then I can certainly see where the problem would come from.

In case we are not, would you happen to know when the features would drop for general release?

**Reply by squidfunk** [reply]
Card grids are in the '[Goat's Horn](https://squidfunk.github.io/mkdocs-material/insiders/#14000-goats-horn)' funding goal which is at $14,000. When that goal is hit, all features that are tied to it are released as part of the community edition. We're currently at ~$5,500, so there's still some way to go.

**Reply by pieroliviermarquis** [reply]
I think you should make it clearer that it is only available for insiders. I just spent 2 hours trying to make grids work not knowing it wasn't supposed to work. Thanks for the great work.

**Reply by squidfunk** [reply]
How could we make it clearer? There's the pumping Insiders heart and the version tag which says that it's available as part of Insiders. Also, all conventions are explained here: https://squidfunk.github.io/mkdocs-material/philosophy/#conventions

**Reply by pieroliviermarquis** [reply]
Yes, but most people will skim through documentations and not read the entire thing before coding. A pumping heart is not a symbol commonly associated with something not available and so is the word experimental which usually refers to something being tested. If you wrote something such as "Only available to Insiders" on the line with the pumping heart, it would have been immediately obvious.

**Reply by squidfunk** [reply]
I went ahead and changed ❤️ __Insiders__ to ❤️ __Sponsors only__ in https://github.com/squidfunk/mkdocs-material/commit/6bc1886251f30f6b27c69e4d46c99decc8554afe – let's see if that's better 😊

**Reply by pieroliviermarquis** [reply]
Yes, that makes a lot of sense. Thanks.
